bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -423,3 +426,63 @@ spec:
         operator: "Exists"
       - key: "node.kubernetes.io/network-unavailable"
         operator: "Exists"
+---
+
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-master-metrics
+  namespace: openshift-ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      RBAC Proxy to expose metrics over HTTPS
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ovnkube-master-metrics
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: ovnkube-master
```

Comment:
@juanluisvaladas ovnkube-master-metrics? probably shouldn't be the same app label as the actual master.